app/assets/stylesheets/pin-show.css.scss

```diff
@@ -54,6 +56,15 @@ button.back{
   }
 }
 
+#numLikes{
```

Comment:
instead of a class? i dont mind, thought using id wouldn't duplicate code, because there is 2 different number of likes